extensions/2.0/Vendor/EXT_structural_metadata/README.md

```diff
@@ -11,40 +13,522 @@
+>           }
+>         }
+>       ]
+>     }
+>   }
+> }
+> ```
+>
+> _Primitive_
+>
+> ```jsonc
+> {
+>   "primitives": [
+>     {
+>       "attributes": {
+>         "POSITION": 0,
+>         "TEXCOORD_0": 1
+>       },
+>       "indices": 2,
+>       "material": 0,
+>       "extensions": {
+>         "EXT_structural_metadata": {
+>           "propertyTextures": [0]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
+
+A `propertyTexture` object extends the glTF [`textureInfo`](../../../../specification/2.0/schema/textureInfo.schema.json) object. `texCoord` specifies a texture coordinate set in the referring primitive.
+
+The `properties` map specifies the texture channels providing data for available properties. An array of integer index values identifies channels. Channels of an `RGBA` texture are numbered 0–3 respectively, although specialized texture formats may allow additional channels. All values are stored in linear space.
+
+> **Example:** A property texture for 2D wind velocity samples. The "speed" property values are stored in the red channel. The "direction" property values are stored as a unit-length vector, with X/Y components in the green and blue channels. Both properties are indexed by UV coordinates in a `TEXCOORD_0` attribute.
+>
+> ```jsonc
+> // Root EXT_structural_metadata extension:
+> {
+>   "propertyTextures": [
+>     {
+>       "class": "wind",
+>       "index": 0,
+>       "texCoord": 0,
+>       "properties": {
+>         "speed": {
+>           "channels": [0]
+>         },
+>         "direction": {
+>           "channels": [1, 2]
+>         }
+>       }
+>     }
+>   ]
+> }
+
+Multiple channels can be used to represent individual bytes of larger data types: The values from the selected channels are treated as unsigned 8 bit integers, and represent the bytes of the actual property value, in little-endian order.
+
+> **Example:** 
+> 
+> If a property is defined to store (single) `FLOAT32` components, then these values can be stored in the 4 channels of a property texture. The raw bits of the property value can be computed as 
+> ```
+> vec4<uint8> rgba = texture(sampler, coordinates);
+> uint8 byte0 = rgba[channels[0]];
+> uint8 byte1 = rgba[channels[1]];
+> uint8 byte2 = rgba[channels[2]];
+> uint8 byte3 = rgba[channels[3]];
+> int32 rawBits = byte0 | (byte1 << 8) | (byte2 << 16) | (byte3 << 24);
+> ```
+> 
+> If a property has the type `VEC2` with `UIN16` components, or an array with a fixed length of 2 and `UINT16` components, then the respective property can be represented with 4 channels as well:
+> ```
+> vec4<uint8> rgba = texture(sampler, coordinates);
+> uint8 byte0 = rgba[channels[0]];
+> uint8 byte1 = rgba[channels[1]];
+> uint8 byte2 = rgba[channels[2]];
+> uint8 byte3 = rgba[channels[3]];
+> value[0] = byte0 | (byte1 << 8);
+> value[1] = byte2 | (byte3 << 8);
+> ```
+
+Texture filtering must be `9728` (NEAREST), `9729` (LINEAR), or undefined, for any texture object referenced as a property texture.
```

Comment:
After struggling with https://github.com/CesiumGS/cesium/pull/10111 I added a note in `EXT_mesh_features` that texture values must be encoded in linear space. Something similar should be added here.

https://github.com/CesiumGS/glTF/commit/0db466b9d32ab458affbe34379959489a46921a5

Reply:
Above, at https://github.com/CesiumGS/glTF/pull/52/files/399f988f41d8a9d2160d3c219088523ec90f483c#diff-7a962affc914907292f8f9f93402e72b6b49599a6bb34b57592955050339c7a1R463 , it already says 

> All values are stored in linear space.

but it may be worth mentioning here again.

Reply:
Ah, I might have just missed that. No need to mention it in two places. Though could you update the wording from `All values are stored in linear space` to `Texture values must be encoded with a linear transfer function.`